### PR TITLE
[WIP] colorize warn output in s/P/CheckInstall

### PR DESCRIPTION
**Pull request by kadoban**
Working on prettying up more log/warn/error output in various places.

* [ ] Any changes that could be relevant to users have been recorded in the ChangeLog.md
* [ ] The documentation has been updated, if necessary.

### CODE REVIEW

File: src/Path/CheckInstall.hs [hunk 1]
```diff
@@ -25,32 +27,29 @@ warnInstallSearchPathIssues destDir installed = do
...
-                            , "\" executable found on the PATH environment variable is "
-                            , T.pack exePath
-                            , ", and not the version that was just installed."
-                            ]
-                        logWarn $ T.concat
-                            [ "This means that \""
-                            , exe
-                            , "\" calls on the command line will not use this version."
-                            ]
+                        prettyWarnL
```
Comment (mgsloan) [inline src/Path/CheckInstall.hs]: Would be best to combine these two into a single warn call so that it just says "Warning:" once

**Comment by mgsloan** [comment]
LGTM, just the one comment

File: src/Path/CheckInstall.hs (hunk 1 above)
Comment (kadoban) [reply src/Path/CheckInstall.hs]: Ah, yes nice catch thanks. Will change.

**Comment by mgsloan** [comment]
@kadoban Is this one good to merge?  Looks ready to me.

**Comment by kadoban** [comment]
@mgsloan Ah yeah it actually is AFAIK. I was going to add more colorizing to it, but I'll just start another for other files.

**Comment by mgsloan** [comment]
Rad, thanks!

**Pull request merged**